src/components/Plants/PlantingSiteDetails.tsx

```diff
@@ -27,6 +26,11 @@ export type PlantingSiteZone = {
   plots: PlantingSitePlot[];
 };
 
+export type PlantingSiteSpecies = {
```

Comment:
This has the same attributes than Population, could we use that type?

Reply:
good catch 👍

Reply:
fixed